### PR TITLE
Optimze conan better

### PR DESCRIPTION
**Pull request by Presburger**
issue: #779

### CODE REVIEW

File: benchmark/hdf5/benchmark_knowhere_float_qps.cpp [hunk 1]
```diff
@@ -194,7 +194,12 @@ class Benchmark_knowhere_float_qps : public Benchmark_knowhere, public ::testing
 
 TEST_F(Benchmark_knowhere_float_qps, TEST_IVF_FLAT_NM) {
 #ifdef USE_CUDA
+#ifdef KNOWHERE_WITH_RAFT
+    index_type_ = knowhere::IndexEnum::INDEX_RAFT_IVFFLAT;
+#else
+
     index_type_ = knowhere::IndexEnum::INDEX_FAISS_GPU_IVFFLAT;
```
Comment (matrixji) [inline benchmark/hdf5/benchmark_knowhere_float_qps.cpp:201]: Do we possibly do all benchmarks together, enable RFAT does not mean FAISS_GPU is not possible, and also enabling GPU not means CPU is not possible.
Maybe another PR could handle it.

Comment (Presburger) [reply benchmark/hdf5/benchmark_knowhere_float_qps.cpp:201]: @matrixji in the future work, we will Abandon support for GPU in Faiss. raft looks better.

**Comment by Presburger** [comment]
@matrixji  hello, I read you blog about the benchmark of faiss ivf & raft ivf, have you try the latest version of master? in my option, raft maybe much better than faiss when use multi thread query.

**Comment by matrixji** [comment]
> @matrixji hello, I read you blog about the benchmark of faiss ivf & raft ivf, have you try the latest version of master? in my option, raft maybe much better than faiss when use multi thread query.

Not finished some tests with massive threads (which planned later). Currently, by reading some slides from GTC, I think RAFT is more suitable for online service, and the original faiss seems more suitable for batch work. And RAFT also has advantages in building an index, especially for a large index.

**Comment by matrixji** [comment]
:-) Glad to see that hdf5 is resolved in cmake right now.

**Comment by Presburger** [comment]
> > @matrixji hello, I read you blog about the benchmark of faiss ivf & raft ivf, have you try the latest version of master? in my option, raft maybe much better than faiss when use multi thread query.
> 
> Not finished some tests with massive threads (which planned later). Currently, by reading some slides from GTC, I think RAFT is more suitable for online service, and the original faiss seems more suitable for batch work. And RAFT also has advantages in building an index, especially for a large index.

yes, RAFT is better for milvus.

**Pull request merged**